app/src/main/res/layout/topic_overview_fragment.xml

```diff
@@ -22,7 +22,10 @@
       <ImageView
         android:id="@+id/topic_thumbnail_image_view"
         android:layout_width="0dp"
-        android:layout_height="200dp"
+        android:layout_height="210dp"
+        android:paddingTop="22dp"
+        android:foreground="@drawable/thumbnail_gradient"
```

Comment:
This works on API 23 only, can we test this on devices with lower API and appropriate support for that.

Reply:
Fixed this by creating a Framelayout on top of imageview, and added gradient to framelayout.